mappings/net/minecraft/block/RedstoneWireBlock.mapping

```diff
@@ -7,8 +7,15 @@ CLASS net/minecraft/class_2457 net/minecraft/block/RedstoneWireBlock
 	FIELD field_11439 WIRE_CONNECTION_WEST Lnet/minecraft/class_2754;
 	FIELD field_11440 WIRE_CONNECTION_NORTH Lnet/minecraft/class_2754;
 	FIELD field_24413 DOT_SHAPE Lnet/minecraft/class_265;
+	FIELD field_24414 LINE_SHAPE Ljava/util/Map;
+	FIELD field_24415 LINE_SIDE_SHAPE Ljava/util/Map;
```

Comment:
hmm, why is a "shape" a map than a voxel shape? If the map key is like axis then name it like `SHAPES` (plural)